server/routes.js

```diff
@@ -9,6 +9,7 @@ var errors = require('./components/errors');
 module.exports = function(app) {
 
   // Insert routes below
+  app.use('/api/posts', require('./api/post'));
```

Comment:
Slightly nitpicking but not sure this is the best endpoint name. Would much rather have a blog endpoint and have posts under that.